Heejung/Lv2/다리를 지나는 트럭.js

```diff
@@ -0,0 +1,25 @@
+function solution(bridge_length, weight, truck_weights) {
+  const running = new Array(bridge_length).fill(0); // 건너는 중인 트럭
+  const waiting = [...truck_weights]; // 대기 중인 트럭

```

Comment:
```suggestion
  const waiting = truck_weights; // 대기 중인 트럭
```
두 배열을 모두 쓰지 않을거면 변수 초기화를 이렇게 하는게 메모리 자원을 덜 써요.
깊은참조와 얕은 참조에 대해 알아보시면 좋을거같아요.